core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -271,6 +276,17 @@ private void registerJsonRpcRoute(
               routingContext.response().putHeader(CONTENT_TYPE, JSON_UTF_8).end(jsonRpcResponse);
             },
             false);
+
+    router
+        .post(JSON_RPC_PATH + "/filecoin")
```

Comment:
(me think) putting `router
        .post(JSON_RPC_PATH + "/filecoin") ` before `router.post(JSON_RPC_PATH)` would force vertx to register and call handler in correct order. (or may be not since we are not using `/*`)

Reply:
done